### PR TITLE
481-feat: Add aws devops course page

### LINKED ISSUE #481: Add "AWS DevOps" Course Information to RS School Website
Please add information about the new course, "AWS DevOps," to the RS School website. All available course information can be found at: [AWS DevOps Course Details](https://wearecommunity.io/events/aws-devops-course).

### PR DESCRIPTION
**Pull request by Wystov**
## What type of PR is this? (select all that apply)

- [x] 🍕 Feature

## Description

add aws devops course page.

## Related Tickets & Documents

- Related Issue #481 
- Closes #481

## Added/updated tests?

- [x] 🙅‍♂️ No, because they aren't needed

## [optional] What gif best describes this PR or how it makes you feel?

IMG_4881

## Summary by CodeRabbit

- **New Features**
	- Introduced a new course on AWS DevOps, expanding educational offerings.
	- Added a dedicated page for the AWS DevOps training program with comprehensive course details.
	- Enhanced breadcrumb navigation to include AWS DevOps.
	- Integrated a new Discord link for AWS DevOps course discussions.
	- Updated course data structures to include prerequisites and learning outcomes for AWS DevOps.
	- Increased maximum height of a component for better content display.

- **Bug Fixes**
	- Adjusted visual presentation of components for improved user experience.

- **Documentation**
	- Expanded course titles and descriptions to reflect new AWS DevOps content.

### CODE REVIEW

File: src/app/services/data/courses.data.ts [hunk 1]
```diff
@@ -152,4 +152,20 @@ export const courses: Course[] = [
...
+    title: COURSE_TITLES.AWS_DEVOPS,
+    iconSrc: aws,
+    iconSmall: awsDevSmall,
+    secondaryIcon: awsSecondary,
+    startDate: 'Sep 23, 2024',
+    language: ['en'],
+    mode: 'online',
+    detailsUrl: `/${ROUTES.COURSES}/${ROUTES.AWS_DEVOPS}`,
+    enroll: 'https://wearecommunity.io/events/aws-devops-course',
+    backgroundStyle: {
```
Comment (ansivgit) [inline src/app/services/data/courses.data.ts:166]: Please, change color to AWS color

**Review by ansivgit — APPROVED** [review]

File: src/app/services/data/courses.data.ts (hunk 1 above)
Comment (Wystov) [reply src/app/services/data/courses.data.ts:166]: Done

**Review by Quiddlee — APPROVED** [review]

File: src/widgets/training-program/training-program.data.tsx [hunk 2]
```diff
@@ -284,4 +285,42 @@ export const contentMap: ContentMap = {
...
+      <Paragraph key="aws devops 01">
+        If you are looking for an entry point to kickstart your IT career as a DevOps engineer,
+        then this AWS course challenge is what you need.
+      </Paragraph>,
+      <Subtitle
+        key="aws devops 02"
+        text="Showcase your level of expertise and join this expert-led program to:"
+      />,
+      <List
+        key="aws devops 02"
```
Comment (SpaNb4) [inline src/widgets/training-program/training-program.data.tsx:300]: Same key

**Review by Yelunina — APPROVED** [review]

File: src/widgets/training-program/training-program.data.tsx (hunk 2 above)
Comment (Wystov) [reply src/widgets/training-program/training-program.data.tsx]: Changed, thank you!

**Review by SpaNb4 — APPROVED** [review]

**Pull request merged**